modules/core/src/main/java/org/apache/ignite/cache/query/TextQuery.java

```diff
@@ -79,6 +82,19 @@ public TextQuery(String type, String txt) {
         setText(txt);
     }
 
+    /**
+     * Constructs query for the given search string.
+     *
+     * @param type Type.
+     * @param txt Search string.
+     * @param limit Limit.
```

Comment:
This is a public API class.
Let's make javadoc more verbose for 'limit' param at least.